extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -633,13 +633,15 @@ private void handleHibernateORMWithNoPersistenceXml(
                 hibernateConfig.jdbc.timezone.ifPresent(
                         timezone -> desc.getProperties().setProperty(AvailableSettings.JDBC_TIME_ZONE, timezone));
 
-                hibernateConfig.jdbc.statementFetchSize.ifPresent(
-                        fetchSize -> desc.getProperties().setProperty(AvailableSettings.STATEMENT_FETCH_SIZE,
-                                fetchSize.toString()));
+                if (hibernateConfig.jdbc.statementFetchSize.isPresent()) {
+                    desc.getProperties().setProperty(AvailableSettings.STATEMENT_FETCH_SIZE,
+                            String.valueOf(hibernateConfig.jdbc.statementFetchSize.getAsInt()));
+                }
```

Comment:
Maybe instead go with the original but slightly modified:
```suggestion
                hibernateConfig.jdbc.statementFetchSize.ifPresent(
                        fetchSize -> desc.getProperties().setProperty(AvailableSettings.STATEMENT_FETCH_SIZE,
                                Integer.toString(fetchSize));
```
Smaller diffs are usually preferred.